lalrpop/src/lr1/build/test.rs

```diff
@@ -281,6 +281,7 @@ T: () = {
 fn issue_144() {
     let _tls = Tls::test();
 
+    #[allow(clippy::needless_raw_string_hashes)] // Fix merged for next clippy release, after 1.72
```

Comment:
I vote against the comment that will get stale really quickly.

Reply:
I'm confused? Yes it will get stale but the point is that, at that point, this allow can be removed. Clippy sometimes has false positives that get fixed in a subsequent version so we may as well keep track of false positives and when they can be removed.(Here the fix has already been merged into master)

Reply:
Oh, I misunderstood.  I thought you were saying this allow wasn't needed yet, but would be needed after the clippy update to 1.72.  It's exactly the other way around.

In that case, maybe the comment would be more clear if it specified the expected action of removing this after 1.72?